debian/changelog

```diff
@@ -1,12 +1,48 @@
-ubuntu-advantage-tools (25.0) UNRELEASED; urgency=medium
+ubuntu-advantage-tools (25.0~beta) UNRELEASED; urgency=medium
 
-  * d/control: lint warning update debhelper version 9->10 avoid deprecation
+  * d/control:
     - add po-debconf dependency and fix lintian not-using-po-debconf and
       untranslatable-debconf-templates
     - add ${misc:Depends} dep to ubuntu-advantage-pro to fix lintian
-      debhelper-but-no-misc-depends
-  * d/rules: set fix lintian warning extra:Depends even if empty
-  * Open 25.0 release for active development
+      debhelper-but-no-misc-depends (GH: #1024)
+  * d/rules:
+    - drop --with systemd fix build-depends-on-obsolete-package
+    - set fix lintian warning extra:Depends even if empty
+  * d/postrm
+    - Add more gpg keys to be deleted in postrm for Xenial+ support
+  * d/postinst:
+    - do not unconfigure non-trusty esm. no series in apt filenames (GH: #1170)
+    - check if esm is already enabled (GH: #1095)
+  * New upstream release 25.0:
+    - Do not uninstall additionalPackages or livepatch when disabling services
```

Comment:
Just a minor nit, I think this behavior is applied to all services and not only livepatch

Reply:
My mistake here, it says `or` livepatch, so it is correct